site/js/index.js

```
@@ -1,223 +1,223 @@
-    } else {
-        onDataRetrieved(JSON.parse(localStorage.getItem("_wfdata")))
+  validateData();
+
+  const data = await fetch(`./data/info.json?${time}`).then(response => response.json());
+  const { hash } = data;
+  console.log('./data/info.json', data);
+  if (hash !== JSON.parse(localStorage.getItem('_wfinfo')).hash || !localStorage.getItem('_wfdata')) {
+    console.log('new hash found or missing local data, re-download data');
+    localStorage.setItem('_wfinfo', JSON.stringify(data));
+    const all = await fetch(`./data/all.slim.json?${time}`).then(response => response.text());
+    if (all && all.isJSON()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)